tests/Hedgehog.CSharp.Tests/Hedgehog.CSharp.Tests/Hedgehog.CSharp.Tests.csproj

```diff
@@ -0,0 +1,9 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This project won't load on Visual Studio 2015. And there's #108 regarding Visual Studio 2017 😕